internal/pkg/manifest/testdata/backend-svc-nohealthcheck.yml

```diff
@@ -29,4 +29,6 @@ exec: true     # Enable running commands in your container.
 # You can override any of the values defined above by environment.
 #environments:
 #  test:
-#    count: 2               # Number of tasks to run for the "test" environment.
\ No newline at end of file
+#    count: 2               # Number of tasks to run for the "test" environment.
+#    deployment:            # deployment strategy for the "test" environment.
+#       rolling: 'recreate' # key is rolling deployment and value is deployment strategy name i.e. either default or recreate.
```

Comment:
Do we have to update these in test data?

Reply:
Yes we need to as test manifest by default has those lines in it when they create new manifest.

Reply:
These testdata doesn't test that unfortunately. 

They are data consumed by the integration test whose purpose would be to validate the template, not to validate the default manifest. 

I think in this case then we can remove all these lines in `testdata`